django/utils/crypto.py

```diff
@@ -17,7 +18,11 @@
 
 # Use the system PRNG if possible
 try:
-    random = random.SystemRandom()
+    if sys.version_info >= (3, 6):
+        import secrets
+        random = secrets.SystemRandom()
+    else:
+        random = random.SystemRandom()
```

Comment:
In the long run I think we should deprecate `get_random_string` in favor of similar functions provided by the `secrets` module.

I didn't check whether there was a sensible transition plan to make use of `secrets` on Python 3.6 while still supporting older versions.

Since this PR adds complexity without changing the behavior, I don't think it's on the right track.